buildpipeline/pipeline.json

```diff
@@ -564,7 +564,8 @@
         "TreatWarningsAsErrors": "false"
       },
       "BuildParameters": {
-        "PB_ConfigurationGroup": "Release"
+        "PB_ConfigurationGroup": "Release",
+        "PB_PublishType": "myget-blob-versions-msdl"
```

Comment:
"-msdl" doesn't mean anything in the 2.0.0 branch.  You can remove that. I think "myget" will cause the symbol packages to be uploaded to myget and the failing pdb conversion.

Reply:
This is only passing these params to the feed publish job, not to the symbol publish job

Reply:
Yes, I see that now. You still don't need the -msdl.